person.rb

```diff
@@ -37,6 +40,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
+
   private
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement for this project, The `add_rental` method takes `book`  and `date` as  parameters, and  `person` itself is sent as a parameter to create the new rental achieving the 'has-many' association. Something like so: 

```ruby
  def add_rental(book, date)
    Rental.new(date, self, book)
  end
```

Please kindly fix that.